src/en/proposals/emogarbage-gods-gamemode.md

```diff
@@ -0,0 +1,134 @@
+Be careful, however, as the results of preaching lessen after someone has be preached to.
+Additionally, this can quickly make you a target of security and other sects. 
+It's best saved for low-level disciples.
+
+Based on a god's theme, you can sacrifice various items to raise their favor. 
+A god of hunting may want animals, greed may want valuables, and a god of silliness may want the entire clown.
+
+Alongside these methods, each god also has unique methods of raising their divinity. 
+These can range from cooking meals, to setting things on fire, to slipping people with soap. 
+These are by far the most effective methods, and encourage disciples to get out into the station and cause some mischief.
+
+## Abilities
+Gods all have unique abilities based on their themes. However, they all also have a few generic ones:
+
+>**Divine Word (Cost: None)**: Sends a private message to your prophet.
+
+>**Wrath (Cost: None)**: Usable on any disciple: deals a small amount of damage and stuns them.
+
+>**Smite (Cost: None)**: Instantly obliterates your disciple, returning your **ascension levels**. 
+> Can only be used on living disciple who is not currently taking damage.
+
+>**Bless (Cost: Low)**: Gives a temporary buff to a target disciple, healing them, increasing their movement speed, and giving them armor.
+
+These abilities are more specialized per-god and can be applied to temples:
+
+>**Give Artifact (Cost: Low)**: Creates a god-specific equipment item at a target altar.
+> If sacrificed at another god's altar, grants a large divinity buff.
+
+>**Imbue Temple (Cost: Medium)**: Permanently imbues a target temple with godly energy.
+> Any disciples of other gods who are nearby will find themselves weakened in its presence.
+
+The following abilities are only unlocked with progressively higher **ascension levels**:
+
+>**Divine Rage (Cost: Medium)**: Does a god-specific attack at a target location. 
+> Cannot be performed near other god's temples. 
+
+>**Inspiration (Cost: Medium)**: Buff all disciples in a radius with a strengthened version of "Bless."
+
+>**Catastrophe (Cost: High)**: Creates a highly destructive attack at a given area. 
+> Can be used near non-imbued temples.
+
+>**Ascension (Cost: Max)**: Ascends a god, smiting other gods and followers. 
+> Ends the gamemode.
+
+Gods finally have one last "ability," creating disciples.
+Any player at a temple or altar can offer themselves as a disciple to a god.
+You can accept their offer at the cost of an **ascension level**. 
+
+It's important to be particular about who you accept, as dead disciples do not return ascension levels (only smites).
+
+## Secular Interference
+All sects are antagonists and thereby subject to the whims of security players and validhunters alike.
+A disciple (not prophet) can be forcefully deconverted with a mindshield, removing them from the sect.
```

Comment:
there should be alternatives to mindshields imo, otherwise it'll turn into "cargo is always the critical target" much like how revs was